### DISCUSSION TITLE
The tenantCode is default, please enable TenantConfig#isDefaultTenantEnabled

### DISCUSSION THREAD

**Discussion by 18065013** [apache/dolphinscheduler#16607] [Q&A]
I deployed dolphinscheduler with docker-compose. throw error log, when i run test task. how to resolve it? 
task inclue 2 shell steps , echo "1" and echo "2"

<img width="912" alt="1726050428983" src="https://github.com/user-attachments/assets/64936c5f-e16c-4e88-ba41-e604858f8e23">

**Comment by 18065013** [comment]
i resovled, edit application.yaml default-tenant-enabled: true and restart.

**Reply by parkour99** [reply]
Why can't the worker select other groups? I don't want to enable this parameter. Moreover, how can I enable this parameter under docker-compose?
